### DISCUSSION TITLE
Perspective Change and Multiple Chat Panels Warning

### DISCUSSION THREAD

**Discussion by ferhatcelebi** [Genuitec/Copilot4Eclipse-community#6] [Help]
Hello all,

Changing perspective from Java EE to Java affects Chat Panel. The panel disappears after a change in perspective. It is a small but fixable problem.
<img width="569" alt="ChatPanel_PerspectiveChange" src="https://github.com/user-attachments/assets/364ed11a-7cf5-477c-9cec-7d36f801448c">

**Comment by wayneparrott** [comment]
Thx for sharing. 

I've added your comment to our issue tracking system. We are working smoothing out perspective changes to avoid this annoying quirk.
